python/ql/lib/change-notes/2022-11-14-grouped-exceptions-breaking.md

```diff
@@ -0,0 +1,4 @@
+---
+category: breaking
```

Comment:
With the introduction of `ExceptionHandler`, I guess this change is no longer breaking -- from my point of view `fix` would be sufficient